### DISCUSSION TITLE
Custom Rules are disappearing from /opt/so/rules/nids/all.rules

### DISCUSSION THREAD

**Discussion by slelanchezian** [Security-Onion-Solutions/securityonion#10723] [General]
Custom rules added in the file /opt/so/rules/nids/all.rules are getting removed automatically after a day. Is this the default behavior. 

How to add permanent rules?

**Comment by InfosecGoon** [comment]
That file is rebuilt automatically by Salt.

To add new rules: https://docs.securityonion.net/en/2.3/local-rules.html
